app/views/search/_result_geo.html.erb

```diff
@@ -8,9 +8,10 @@
   </div>
 
   <% if result_geo['contributors'] %> 
-    <p class="authors truncate-list">
-      <%= render partial: 'shared/authors', locals: { contributors: result_geo['contributors'] } %>
-    </p>
+    <span class="sr">Contributors: </span>
```

Comment:
I think it would be a bit of a refactor to do this, so I don't think we should in this change... but if we changed `result_geo` to just be `result` wherever it came from we'd be able to remove some duplication and move the entire contributors logic into the shared partial instead of having geo/normal variants that are identical.